### PR TITLE
[TACHYON-131]Add FAQ page to tachyon webpage

### PR DESCRIPTION
**Pull request by kvaishnav**
Add initial FAQ page. This page can be updated every few months with more questions/answers.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+title: Tachyon FAQ
+---
+
+## Tachyon Frequently Asked Questions.
+
+1. What is Tachyon?
+[Tachyon](http://tachyon-project.org/index.html) is an in-memory distributed, reliable storage system enabling data sharing across cluster at memory speed. Data reliability is achieved by using lineage concept. In memory speed is acheived by caching working file sets, avoiding performing disk reads.
+
+2. What platforms and Java versions does Tachyon run on?
+Currently Tachyon has been tested with Oracle Java 1.7 running on various distributions of Linux.
```
Comment (calvinjia) [inline docs/FAQ.md]: Isn't Tachyon currently compiled with java 1.6?

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Yes, Once Tachyon is installed, please login into [http://master_ip:19999](http://master_ip:19999)
+
+4. If I add new nodes to the cluster, will tachyon rebalance/move the cached blocks to the newly added nodes in order to balance disk space utilization between the nodes?
+No, Currently re-balancing of data blocks in not available.
+
+5. How do you achieve Tachyon master redundancy?
+We use Tachyon Master/Stand-by architecture. Also, zookeeper can be used to pick a Tachyon master.
+
+6. What is underlayer FS and how many Underlayer FS are supported?
+Underlayer FS is a general distributed filesystems on which tachyon works seamlessly used as long term storage for fault tolerant and to re-compute missing data. Tachyon inherently is not a filesystem.
```
Comment (calvinjia) [inline docs/FAQ.md]: Is this still true with hierarchical storage?

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Yes, Tachyon can be run in a stand-alone mode.Instructions are available [here](http://tachyon-project.org/Running-Tachyon-Locally.html)
+
+11. Can I run Tachyon in a cluster mode?
+Yes, Tachyon can be run in a cluster mode. Instructions are available [here](http://tachyon-project.org/Running-Tachyon-on-a-Cluster.html)
+
+12. Does Tachyon always needs HDFS?
+No, Tachyon runs on different underlayer storage systems like HDFS, Amazon S3, GlusterFS. Instructions on configuration is [available](http://tachyon-project.org/Setup-UFS.html).
+
+13. Does tachyon work along side with other Frameworks?
+Yes, Tachyon works with [hadoop](http://hadoop.apache.org/), [Spark](http://spark.apache.org/), [HBase](http://hbase.apache.org/), [Flink](http://flink.apache.org/), Shark
```
Comment (calvinjia) [inline docs/FAQ.md]: Maybe these should link to the respective frameworks on tachyon pages?

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+	[Video 2](https://www.youtube.com/watch?v=wJ2ocPoj9rs)
+
+15. Are there any meetups for tachyon?
+Yes, We also have started meetup group for Tachyon and please join [http://www.meetup.com/Tachyon/](http://www.meetup.com/Tachyon/). Stay Tuned for upcoming meetup.
+
+16. What license is Tachyon under?
+Tachyon is under the Apache 2.0 license.
+
+17. I am a beginner and How can I contribute to Tachyon?
+Thank you for the interest to contribute. Please look into [Startup-Tasks-for-New-Contributors](http://tachyon-project.org/master/Startup-Tasks-for-New-Contributors.html).
```
Comment (calvinjia) [inline docs/FAQ.md]: This page was recently moved into a subsection of the Contributing-To-Tachyon page.

**Comment by calvinjia** [comment]
Thanks for the patch, an FAQ should be very helpful! Is there a specific ordering to the questions? It might make sense to group them together and provide a link to each section? Currently the FAQ is not very large, but it may end up becoming difficult to navigate.

**Comment by haoyuan** [comment]
@kvaishnav @calvinjia 

This is a good start. I just merged this. I will do a pass on this.

File: docs/FAQ.md (hunk 2 above)
Comment (rvesse) [inline docs/FAQ.md]: Zookeeper is required for Master Fault Tolerance, yes it does pick the master but without it there is no fault tolerance

File: docs/FAQ.md (hunk 3 above)
Comment (rvesse) [inline docs/FAQ.md]: Doesn't behaviour for 9 depend on the user requested behaviour?  E.g. `MUST_CACHE` vs `THROUGH`

**Comment by haoyuan** [comment]
Thanks @rob for the comments. I have done a pass earlier. Here is the most recently one: https://github.com/amplab/tachyon/blob/master/docs/FAQ.md

**Comment by rvesse** [comment]
Updated versions LGTM

**Pull request merged**